packages/driver/cypress/e2e/e2e/origin/cookie_behavior.cy.ts

```diff
@@ -1,34 +1,6 @@
-describe('Cookie Behavior with experimentalSessionAndOrigin=true', () => {
-  const makeRequest = (
-    win: Cypress.AUTWindow,
-    url: string,
-    client: 'fetch' | 'xmlHttpRequest' = 'xmlHttpRequest',
-    credentials: 'same-origin' | 'include' | 'omit' | boolean = false,
-  ) => {
-    if (client === 'fetch') {
-      // if a boolean is specified, make sure the default is applied
-      credentials = Cypress._.isBoolean(credentials) ? 'same-origin' : credentials
-
-      return win.fetch(url, { credentials })
-    }
-
-    return new Promise<void>((resolve, reject) => {
-      let xhr = new XMLHttpRequest()
-
-      xhr.open('GET', url)
-      xhr.withCredentials = Cypress._.isBoolean(credentials) ? credentials : false
-      xhr.onload = function () {
-        resolve(xhr.response)
-      }
-
-      xhr.onerror = function () {
-        reject(xhr.response)
-      }
-
-      xhr.send()
-    })
-  }
+import { makeRequestForCookieBehaviorTests as makeRequest } from '../../../support/utils'
 
+describe('Cookie Behavior with experimentalSessionAndOrigin=true', () => {
```

Comment:
with experimentalSessionAndOrigin=true is implied by being in origin folder
```suggestion
describe('Cookie Behavior', () => {
```

Reply:
I feel like we can drop the flag, but I wonder if something more than `Cookie Behavior` is needed to capture what the spec is actually doing. Then again simpler might be better?